test/inflate.js

```diff
@@ -183,8 +183,18 @@ describe('Inflate with dictionary', function () {
 
   it('spdy dictionary', function () {
     var spdyDict = require('fs').readFileSync(require('path').join(__dirname, 'fixtures', 'spdy_dict.txt'));
+    testInflate(samples, { dictionary: spdyDict }, { dictionary: spdyDict });
+  });
 
-    testInflate(samples, { dictionary: spdyDict }, { dictionary: helpers.spdyDict });
+  it('should throw if directory is not supplied to raw inflate', function () {
+    var dict = new Buffer('abcdefghijklmnoprstuvwxyz');
+    assert.throws(function () {
+      testInflate(samples, { raw: true }, { raw: true, dictionary: dict });
+    });
   });
 
+  it('tests raw inflate with spdy dictionary', function () {
+    var spdyDict = require('fs').readFileSync(require('path').join(__dirname, 'fixtures', 'spdy_dict.txt'));
+    testInflate(samples, { raw: true, dictionary: spdyDict }, { raw: true, dictionary: spdyDict });
+  });
```

Comment:
Could you add tests with all supported dictionary types (array, typed array)? Pako has very high coverage, and it would be nice to keep it high.

Use [`make cover`](https://github.com/nodeca/pako/blob/master/Makefile#L30) to check

Reply:
That should by another PR as it is not related to bug in raw inflate.

Reply:
Added sufficient tests in https://github.com/nodeca/pako/pull/156/commits/b4f1c205efdf998f512cbf5484445fc0b30f8051